uninstall.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ *
+ * This file runs when the plugin in uninstalled (deleted).
+ * This will not run when the plugin is deactivated.
+ * Ideally you will add all your clean-up scripts here
+ * that will clean-up unused meta, options, etc. in the database.
+ *
+ */
+
+// If plugin is not being uninstalled, exit (do nothing)
```

Comment:
:no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).